lib/matplotlib/figure.py

```diff
@@ -182,10 +182,12 @@ def __init__(self, left=None, bottom=None, right=None, top=None,
             The top of the subplots of the figure
 
         *wspace* : 0.2
```

Comment:
Gotcha. Alright I went through and removed them in a bunch of places in that file. I figured while I was there I wouldn't limit myself to just one class.